plugin/pkg/admission/nodeenv/admission.go

```diff
@@ -0,0 +1,117 @@
+package nodeenv
```

Comment:
Wait to do this at the end of the reviews, but I want to pick a different name for this package.